### PR TITLE
Add TSNE jupyter notebook example

### PR DESCRIPTION
**Pull request by inteldimitrius**
# Description
- TSNE run example added

### CODE REVIEW

**Review by napetrov — APPROVED** [review]

File: examples/notebooks/tsne_mnist.ipynb [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "id": "27b99b44",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "x, y = fetch_openml(name='mnist_784', return_X_y=True)"
```
Comment (Alexsandruss) [inline examples/notebooks/tsne_mnist.ipynb:45]: Data source is not referenced (site/authors).

File: examples/notebooks/tsne_mnist.ipynb [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Intel® Extension for scikit-learn. Divergence: 3.178582994547419\n"
+     ]
+    }
+   ],
+   "source": [
+    "embedding_intelex.shape\n",
```
Comment (Alexsandruss) [inline examples/notebooks/tsne_mnist.ipynb:138]: What is the purpose of this line not returning shape to cell output?

File: examples/notebooks/tsne_mnist.ipynb [hunk 3]
```diff
@@ -0,0 +1,328 @@
...
+    "from sklearnex import unpatch_sklearn\n",
+    "unpatch_sklearn()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "8cfa0dba",
+   "metadata": {},
+   "source": [
+    "Training algorithm with original Scikit-learn library for MNSIT dataset"
```
Comment (Alexsandruss) [inline examples/notebooks/tsne_mnist.ipynb:167]:
```suggestion
    "Training algorithm with original Scikit-learn library for MNIST dataset"
```

File: examples/notebooks/tsne_mnist.ipynb [hunk 4]
```diff
@@ -0,0 +1,328 @@
...
+    }
+   ],
+   "source": [
+    "HTML(f'<h2>With scikit-learn-intelex patching you can:</h2>'\n",
+    "     f\"<ul>\"\n",
+    "     f\"<li>Use your Scikit-learn code for training and prediction with minimal changes (a couple of lines of code);</li>\"\n",
+    "     f\"<li>Fast execution training and prediction of Scikit-learn models;</li>\"\n",
+    "     f\"<li>Get the similar quality</li>\"\n",
+    "     f\"<li>Get speedup in <strong>{(time_original/time_opt):.1f}</strong> times.</li>\"\n",
+    "     f\"</ul>\")"
```
Comment (Alexsandruss) [inline examples/notebooks/tsne_mnist.ipynb:303]: image
Github file preview doesn't look nice for this cell

**Review by Alexsandruss — CHANGES_REQUESTED** [review]

File: examples/notebooks/tsne_mnist.ipynb (hunk 1 above)
Comment (inteldimitrius) [reply examples/notebooks/tsne_mnist.ipynb:51]: Done.

File: examples/notebooks/tsne_mnist.ipynb (hunk 3 above)
Comment (inteldimitrius) [reply examples/notebooks/tsne_mnist.ipynb]: Fixed.

File: examples/notebooks/tsne_mnist.ipynb (hunk 4 above)
Comment (inteldimitrius) [reply examples/notebooks/tsne_mnist.ipynb]: Removed.

**Review by Alexsandruss — APPROVED** [review]

**Pull request merged**